Assets/OrderIndependentTransparency/Shaders/LinkedListCreation.cginc

```diff
@@ -16,7 +16,7 @@ RWByteAddressBuffer StartOffsetBuffer : register(u2);
 
 void createFragmentEntry(float4 col, float3 pos, uint uCoverage) {
     //Retrieve current Pixel count and increase counter
-    uint uPixelCount = FLBuffer.IncrementCounter();
+    uint uPixelCount = FLBuffer.IncrementCounter()+1;
```

Comment:
i leave first one empty for safer debug, otherwise it's easy to make a infinite loop and make my computer crash